frontend/src/pages/Home.jsx

```diff
@@ -31,34 +52,355 @@ class Home extends Component {
           </Typography>
           <Button 
             variant="contained" 
-            color="primary"
             style = {{
-              marginRight: 10
+              marginRight: 10,
+              color: "#0F1111",
+              background: "#f0c14b",
+              fontWeight: 600
+            }}
```

Comment:
picky: personally I prefer css stylings to be in css files instead of inline styling, but this is not something we need to change at this stage.

Reply:
inline style is easy for material ui framework, putting it into css will require lots of work. But we'll discuss what we should do.